mobile-app/app/screens/AppNavigator/screens/Loans/screens/PaybackLoanScreen.tsx

```diff
@@ -622,145 +457,196 @@ function TransactionDetailsSection ({
-          suffix: selectedPaymentToken.tokenDisplaySymbol
-        }}
-      />
-      <TextRow
-        lhs={translate('screens/PaybackLoanScreen', 'Vault ID')}
-        rhs={{
-          value: vault.vaultId,
-          testID: 'text_vault_id',
-          numberOfLines: 1,
-          ellipsizeMode: 'middle'
-        }}
-        textStyle={tailwind('text-sm font-normal')}
-      />
-      <NumberRow
-        lhs={translate('screens/PaybackLoanScreen', 'Remaining loan amount')}
-        rhs={{
-          value: BigNumber.max(outstandingBalance.minus(amountToPayInLoanToken), 0).toFixed(8),
-          testID: 'text_resulting_loan_amount',
-          suffixType: 'text',
-          suffix: displaySymbol
-        }}
-      />
-      {resultingColRatio.isLessThan(0)
-        ? (
           <TextRow
-            lhs={translate('screens/PaybackLoanScreen', 'Resulting collateralization')}
+            containerStyle={{
+              style: tailwind('flex flex-row pb-1'),
+              dark: tailwind('bg-gray-800 border-gray-700'),
+              light: tailwind('bg-white border-gray-200')
+            }}
+            lhs={{
+              value: translate('screens/PaybackLoanScreen', 'Min. Col. Ratio'),
+              themedProps: rowStyle.lhsThemedProps,
+              testID: 'lhs_min_col_ratio'
+            }}
+            rhs={{
+              value: `${vault.loanScheme.minColRatio}%`,
+              testID: 'text_min_col_ratio',
+              numberOfLines: 1,
+              ellipsizeMode: 'middle',
+              themedProps: rowStyle.rhsThemedProps
+            }}
+            textStyle={tailwind('text-xs font-normal')}
+          />
+          <NumberRow
+            {...rowStyle}
+            lhs={translate('screens/PaybackLoanScreen', 'Total collateral (USD)')}
             rhs={{
-              value: translate('screens/PaybackLoanScreen', 'N/A'),
-              testID: 'text_resulting_col_ratio'
+              value: vault.collateralValue,
+              testID: 'text_total_collateral_usd',
+              prefix: '$'
             }}
-            textStyle={tailwind('text-sm font-normal')}
-            info={collateralAlertInfo}
+            textStyle={tailwind('text-xs font-normal')}
           />
-        )
-        : (
-          <CollateralizationRatioRow
-            label={translate('screens/PaybackLoanScreen', 'Resulting collateralization')}
-            value={resultingColRatio.toFixed(2)}
-            testId='text_resulting_col_ratio'
-            type='current'
-            minColRatio={new BigNumber(vault.loanScheme.minColRatio)}
-            totalLoanAmount={new BigNumber(vault.loanValue).minus(
-              totalPaybackWithInterest.multipliedBy(loanTokenPrice)
-            )}
-            colRatio={resultingColRatio}
+          <NumberRow
+            {...rowStyle}
+            lhs={translate('screens/PaybackLoanScreen', 'Total loan (USD)')}
+            rhs={{
+              value: vault.loanValue,
```

Comment:
make it 2 decimal places for consistency?

Reply:
Updated!